playbooks/roles/crafty-bot/meta/main.yml

```diff
@@ -6,3 +6,22 @@ dependencies:
     NGINX_PROXY_INTERNAL_PORT: 5000
     NGINX_PROXY_USE_SSL: "{{ USE_SSL }}"
     NGINX_PROXY_SERVER_NAME: "{{ API_HOSTNAME }}"
+    NGINX_PROXY_EXTRA_CONFIG: |
+      # SE-3967 - Prevent Invalid HTTP_HOST headers
+      if ( $host !~* ^{{ all_env_tokens.DJANGO_ALLOWED_HOSTS }}$ ) {
+          return 444;
+      }
+
+      location /flower/ {
```

Comment:
@gabor-boros, I'm not a fan of hardcoding this path, but it can be addressed at a later point, as [this MR](https://gitlab.com/opencraft/dev/crafty-bot/-/merge_requests/4) has already been reviewed by @nizarmah and merged.

Reply:
Ah sorry about that, I lack the experience with flower to know which practices are preferred and which aren't 😳